grafana/postgres/PG_Details.json

```diff
@@ -2346,7 +2454,7 @@
         },
         "datasource": {
           "type": "prometheus",
-          "uid": "${ccp_datasource}"
+          "uid": "PDC1078F23EBDF0E5"
```

Comment:
i don't know enough about dashboard. this changed from a var to a constant? that's intended/correct?

Reply:
err, about dashboards. like, in general

Reply:
Yeah when I did the upstream merge of this it messed it up and undid the change to a variable. It was easier to just refix the dashboard like I did for the variable change PR, but there's still a few spots that the old value is there that I don't know how to fix properly.